homeassistant/components/wemo/__init__.py

```diff
@@ -229,7 +231,7 @@ def async_stop_discovery(self) -> None:
             self._stop()
             self._stop = None
 
-    async def discover_statics(self):
+    async def discover_statics(self) -> None:
         """Initialize or Re-Initialize connections to statically configured devices."""
         if self._static_config:
```

Comment:
If you reverse this condition and return, you can outdent below

Reply:
Done. 96d9573